src/components/shared/PanelSearch.css

```diff
@@ -27,13 +27,16 @@
 }
 
 .panelSearchFieldIntroduction.isDisplayed {
-  transition: transform 150ms var(--animation-curve), opacity 150ms ease-in-out;
+  transition: transform 150ms var(--animation-curve), opacity 150ms ease-in-out,
+    visibility 150ms ease-in-out;
```

Comment:
Probably precising `ease-in-out` is not useful for `visibility`, where the transition maps to 1 or 0 :) but it also does no harm.